packages/fast-element/src/behaviors/binding-base.ts

```diff
@@ -1,32 +1,22 @@
 import { IBehavior } from "./behavior";
-import {
-    IGetterInspector,
-    IPropertyChangeListener,
-    Observable,
-} from "../observation/observable";
+import { IGetterInspector, Observable } from "../observation/observable";
 import { BindingDirective } from "../directives/bind";
 import { DOM } from "../dom";
+import { ISubscriber } from "src/observation/subscriber-collection";
```

Comment:
looks like the autocomplete on your text editor unfortunately did not autocomplete a relative path correctly, I think you want`src/rest/of/path` to be `../rest/of/path`

Reply:
Fixed.